.github/autobump.txt

```diff
@@ -1572,6 +1572,7 @@ squealer
 squid
 sratoolkit
 src
+srgn
```

Comment:
@Homebrew/core do we want to automatically add all software to this list? Maybe we should make a blocklist instead then.

Reply:
Wouldn't it be better to store this information in the formula itself?

Reply:
you mean autobump info?

Reply:
I dont think we have a way of doing it now atm

Reply:
No we don't, but it would seem to be 1. a more relevant place to put it, 2. more scalable (no merge conflicts, no tedious long list of formulas). Should we propose it?

Reply:
Sounds like something we should consider.

Maybe also have an option to specify if bumped by upstream as we removed those formulae in #164770 but we don't have a way to keep track of them. This can lead to us accidentally re-adding them and potentially messing up the upstream release flow. Otherwise, we would probably need another list to record this info to avoid having maintainers manually keep track of it.

Reply:
We should only really make this a formula DSL if we're going to make it possible for third-party taps to use autobump somehow too.

We shouldn't add software automatically here but it does seem reasonable to add it manually.